src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -164,6 +164,68 @@ def _encode_crl_number(backend, crl_number):
     return pp, r
 
 
+_CRL_ENTRY_REASONFLAGS = {
+    x509.ReasonFlags.unspecified: 0,
+    x509.ReasonFlags.key_compromise: 1,
+    x509.ReasonFlags.ca_compromise: 2,
+    x509.ReasonFlags.affiliation_changed: 3,
+    x509.ReasonFlags.superseded: 4,
+    x509.ReasonFlags.cessation_of_operation: 5,
+    x509.ReasonFlags.certificate_hold: 6,
+    x509.ReasonFlags.remove_from_crl: 8,
+    x509.ReasonFlags.privilege_withdrawn: 9,
+    x509.ReasonFlags.aa_compromise: 10
+}
```

Comment:
There's the opposite (int to enum) in x509.py, and there's _CRLREASONFLAGS in backend.py, but the integer mappings are different (even though the names are the same) between how they're encoded in CRLDistributionPoints vs CRLReason inside a CRL entry.